node/pegnet/addresses.go

```diff
@@ -255,6 +302,70 @@ func (Pegnet) selectBalances(q QueryAble, adr *factom.FAAddress) (map[fat2.PTick
 	return balanceMap, nil
 }
 
+// SelectPendingBalances returns a map of all valid PTickers and their associated
+// balances for the given address. If the address is not in the database,
+// the map will contain 0 for all valid PTickers. This works on the pending tx
+func (p *Pegnet) SelectAllBalances() ([]BalancesPair, error) {
+	rows, err := p.DB.Query(`SELECT * FROM pn_addresses;`)
```

Comment:
The only thing that worries me about the '*' is if someone's db has a different column order. That should not happen, but explicitly putting the column names would prevent that. I think this is fine for now

Reply:
i copy-pasted the code from `selectBalances` :smirk_cat: 

i agree that it's not an optimal solution, but this is an issue that should be tackled in #1 so we can write proper queries